pages/reference/base-images/base-images.md

```diff
@@ -66,7 +66,7 @@ In the tags, all of the fields are optional, and if they are left out, they will
 - `<hw>` : raspberrypi3 - The Raspberry Pi 3 device type.
 - `<distro>` : omitted, so it defaults to Debian.
 - `<lang>` : node - the Node.js runtime and npm will be installed
-- `<lang_ver>` : 10.17 - This gives us Node.js version 10.18.x whatever is the latest patch version provided on balenalib
+- `<lang_ver>` : 10.18 - This gives us Node.js version 10.18.x whatever is the latest patch version provided on balenalib
```

Comment:
The 10.18 here is now fixed, but the example image-name a few lines up is still `balenalib/raspberrypi3-node:10.17` ? :eyes: